home/models.py

```diff
@@ -601,6 +613,8 @@ def submit_whatsapp_template(self, previous_revision):
                 and self.is_whatsapp_template == previous_revision.is_whatsapp_template
                 and self.whatsapp_template_image
                 == previous_revision.whatsapp_template_image
+                and self.whatsapp_template_category
+                == previous_revision.whatsapp_template_category
             ):
```

Comment:
Could we factor this out into its own method?

Reply:
Yup, that's a good idea. I separated it out into a method that returns just the fields, for an easy equality check.